.lycheeexclude

```diff
@@ -0,0 +1,166 @@
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/search-field-caps.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-analyze.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-clearcache.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-synced-flush-api.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-flush.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-forcemerge.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-get-settings.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-upgrade.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-update-settings.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-recovery.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-refresh.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-rollover-index.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-segments.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-shards-stores.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-shrink-index.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-split-index.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-stats.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/indices-upgrade.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/search-validate.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/delete-pipeline-api.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/get-pipeline-api.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/grok-processor.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/put-pipeline-api.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/simulate-pipeline-api.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/docs-multi-termvectors.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-nodes-hot-threads.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/secure-settings.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/cluster-nodes-stats.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/search-rank-eval.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/current/search-template.html
+https://www.opensearch.org/guide/en/elasticsearch/painless/master/painless-execute-api.html
+https://www.opensearch.org/guide/en/elasticsearch/reference/master/search-shards.html
+https://www.opensearch.org/elastic-maps-service*
```

Comment:
The origin of this one should be replaced or removed.

Reply:
This is the pain i was facing. As you can see from the PR, I fixed some of them. For the rest of links, I added them into TODO list because I couldn't find out a good replacement them.  Most of them sit in a json file and should be replaced with a right prod url. If I just replace it with a placeholder/temp url. We may lose the tracker of these issues. 
Instead, I add all of them into the TODO section so that we have a record for what we need to fix. We can remove it from the exclude list and when prod url become available (fix the links as well).  What do you think?